src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/MockGoogleDataprocDAO.scala

```diff
@@ -12,12 +12,18 @@ class MockGoogleDataprocDAO extends DataprocDAO {
 
   private val clusters: mutable.Map[String, Cluster] = new TrieMap()
 
+  private val googleID = UUID.randomUUID().toString
+
   override def createCluster(googleProject: String, clusterName: String, clusterRequest: ClusterRequest)(implicit executionContext: ExecutionContext): Future[ClusterResponse] = {
-    val clusterResponse = ClusterResponse(clusterName, googleProject, UUID.randomUUID().toString, "status", "desc", "op-name")
+    val clusterResponse = ClusterResponse(clusterName, googleProject, googleID, "status", "desc", "op-name")
 
     clusters += clusterName -> Cluster(clusterRequest, clusterResponse)
 
     Future.successful(clusterResponse)
   }
 
+  override def deleteCluster(googleProject: String, clusterName: String)(implicit executionContext: ExecutionContext): Future[Unit] = {
+    Future{val clusterResponse = ClusterResponse(clusterName, googleProject, googleID, "status", "desc", "op-name")}
```

Comment:
replace all this with `Future.successful(())`

Reply:
for the record, `()` means "make a new object of type `Unit`"

Reply:
alternately, depending on how real you want this mock to be: delete from the `clusters` map and throw a error if it's not there

Reply:
Technically, "`()` means "return the singleton object of type `Unit`" :)

Reply:
@jmthibault79 I feel like deleting it from the `clusters` map should be enough since the dao does not throw an error in case of a 404

Reply:
ok yes that makes sense